include/deal.II/matrix_free/dof_info.templates.h

```diff
@@ -183,6 +183,98 @@ namespace internal
 
 
 
+    void
+    DoFInfo::get_dof_indices(std::vector<unsigned int> &my_rows,
+                             const unsigned int         cell,
+                             const bool                 apply_constraints) const
+    {
+      const unsigned int n_fe_components = start_components.back();
+      const unsigned int fe_index =
+        dofs_per_cell.size() == 1 ? 0 : cell_active_fe_index[cell];
+      const unsigned int dofs_this_cell = dofs_per_cell[fe_index];
```

Comment:
Could you please add an `AssertIndexRange(cell, n_vectorization_lanes_filled[dof_access_cell]);`